Description: Fix configure.ac for newer autoconf
Author: Ximin Luo <infinity0@debian.org>
Forwarded: no
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/configure.ac
+++ b/configure.ac
@@ -9,6 +9,7 @@
 AC_CONFIG_AUX_DIR([tools])
 AM_INIT_AUTOMAKE([-Wall -Werror -Wno-override foreign no-dependencies])
 AM_MAINTAINER_MODE
+AM_PROG_AR
 
 AC_CONFIG_SRCDIR([src/nsUniversalDetector.h])
 AC_CONFIG_HEADER([chardet-config.h])
